src/shared/components/query/DataTypeFilter.tsx

```diff
@@ -162,80 +162,98 @@ export const DataTypeFilter: FunctionComponent<IDataTypeFilterProps> = props =>
                                         const isZero =
                                             props.studyPerFilter![i] === 0 &&
                                             props.samplePerFilter![i] === 0;
+                                        const isInitialCountZero =
```

Comment:
@gblaih rather than add this stuff here, I think it would make more sense to filter these out of sortedIndicesRef.current before mapping.  that way you don't have to return "nothing".